manifests/image-references

```diff
@@ -122,6 +122,10 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-csi-driver-shared-resource:latest
+  - name: csi-driver-shared-resource-webhook
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-csi-driver-shared-resource-webhook:latest
```

Comment:
You must use this name string in CSO's deployment as SHARED_RESOURCE_DRIVER_WEBHOOK_IMAGE env. var, around here:
https://github.com/openshift/cluster-storage-operator/blob/f356bff281fa449ba8bdda35b68e198c2fcfcf92/manifests/10_deployment.yaml#L114-L115